docs/content/preview/yugabyte-platform/security/create-kms-config/hashicorp-kms.md

```diff
@@ -138,7 +168,13 @@ You can create a new KMS configuration that uses HashiCorp Vault as follows:
     - **Configuration Name** — Enter a meaningful name for your configuration.
     - **KMS Provider** — Select **Hashicorp Vault**.
     - **Vault Address** — Enter the web address of your vault. For example, `http://127.0.0.1:8200`
-    - **Secret Token** — Enter the token you obtained from the vault.
+    - **Authentication Type** — Choose the authentication method to use, Token or AppRole.
+
+        For Token, enter the token you obtained from the vault.
+
+        For AppRole, enter the role ID and corresponding secret ID obtained from the vault. If the vault is configured with namespaces, also enter the namespace used to generate the credentials.
+
+    - **Key Name** - Enter the key name.
```

Comment:
Could we add a variation the following line as well here: `If no key name is specified, it will use the default name key_yugabyte .  If there already exists a key with that name, it will use it, else it will create a key with that name.` Thanks a lot